plans/rpminspect/README.md

```diff
@@ -0,0 +1,48 @@
+# Rpminspect
+
+<!-- SPHINX-START -->
+
+Run [rpminspect] on the current Copr project or Koji build
+
+## Synopsis
+
+```yaml
+plans:
+  import:
+    url: https://github.com/packit/tmt-plans
+    ref: main
+    name: /plans/rpminspect
+```
+
+## Options
+
+`RPMINSPECT_KOJI_BUILD`
+: Run `rpminspect` on the specified Koji build instead of the expected Copr project
+: Note: Not yet implemented
```

Comment:
We should either here or somewhere top-level define how to define these options.

Probably as `env` parameter in .packit.yaml? Btw., do you think we should promote this repository being used primarily through Packit or have it as a shared space for TMT plans in general --> this might require different wording or even implementation...

Reply:
Yes, I do not know how to do that, that's why I didn't fill it in :D

> Probably as `env` parameter in .packit.yaml?

I think there is a TMT native way of doing this as well. I need to play around with it.

> Btw., do you think we should promote this repository being used primarily through Packit or have it as a shared space for TMT plans

This I don't know, we'll have to CC @psss about this. I did float around the idea of having a top-level tmt library to distribute and document these like they are with `ansible` collections, but this still needs to be implemented and evolve.

But packit and fedora-ci workflow would definitely be the main users

Reply:
From the `tmt` point of view the shared library, not limited to `packit` only, makes a bit more sense as importing remote plans is not limited in any way. So, if the code is reusable and not taylored for `packit` only I'd say it can be promoted as a general library. Hm, and if it's not closely tied to `packit` perhaps a question arises whether these would better fit to a location like https://github.com/teemtee/plans/ ? Just brainstorming.

Reply:
For this plan, it mainly uses the artifacts from `testing-farm` (`/var/share/test-artifacts`), which I am not sure if it is equivalent in non-packit jobs. The zuul jobs downstream for example is using just koji builds. I plan to convert the koji tag input for this one as well, but otherwise I don't know how to support other use-cases. Probably would need the ability to chain plans (sort of like making a plan/test a fixture for another) before we could make these plans truly general.

For now maybe we keep these in `packit` until they are polished and production ready, then those could be migrated to another group.

Reply:
Sounds good to me.